app/models/user/canvas/activity.rb

```diff
@@ -0,0 +1,176 @@
+module User
+  module Canvas
+    class Activity
+      include HtmlSanitizer
+
+      attr_accessor :dashboard_sites
+
+      attr_accessor :id, :uid, :title, :created_at, :updated_at, :type,
+        :html_url, :context_type, :course_id, :message, :score,
+        :submission_comments, :classes
+
+      def as_json(options={})
+        {
+          statusDate: max_date&.to_date,
+          statusDateTime: max_date&.to_datetime,
+          emitter: 'bCourses',
+          id: "canvas_#{id}",
+          source: source,
+          sourceName: source,
+          source_url: html_url,
+          title: processed_title,
+          description: summary,
+          type: processed_type,
+          url: html_url
+        }
+      end
+
+      def initialize(attrs={})
+        attrs.each do |key, value|
+          if respond_to?("#{key}=")
+            send("#{key}=", value)
+          else
+            Rails.logger.debug("- - - - - - - - - - -didn't set #{key}")
```

Comment:
@developish are you leaving this logger statement in intentionally?

Reply:
Nope